graphrag/app/supportai/supportai.py

```diff
@@ -414,8 +459,107 @@ def create_ingest(
         data_stream_conn = data_stream_conn.replace(
             "@source_config@", json.dumps(connector)
         )
-    elif ingest_config.data_source.lower() == "local" or ingest_config.data_source.lower() == "remote":
-        pass
+    elif ingest_config.data_source.lower() == "local":
+        # Handle multi-format processing for local files
+        if ingest_config.file_format.lower() == "multi":
```

Comment:
let's use "server" for files stored on graphrag server side and leave "local" to files on client(user) side, which might need to be uploaded to server temporary.